liquidctl/cli.py

```diff
@@ -28,13 +28,17 @@
   --time-off <value>          Time to wait with the LED turned off (seconds)
   --alert-threshold <number>  Threshold temperature for a visual alert (°C)
   --alert-color <color>       Color used by the visual high temperature alert
+  --direction <string>        If the pattern should move forward or backwords. [default: forward]
```

Comment:
There's a typo in the description: s/backwords/backwards/.

Please also add an issue for all drivers that need to be updated to support this new flag (in additional to the old and silly `backwards-` prefixed modes).

Reply:
created #253 for this.